### PR TITLE
PORT-7361 | Mini guide to connect Github repository with SonarQube project

### PR DESCRIPTION
**Pull request by lordsarcastic**
# Description
This mini guide is on the other direction of [connect SonarQube project to service](https://docs.getport.io/build-your-software-catalog/sync-data-to-catalog/code-quality-security/sonarqube/guides/connect-sonar-project-to-service) but does the connection on the GitHub side

## Added docs pages

Please also include the path for the added docs

- Connect GitHub repository with SonarQube project (`docs/build-your-software-catalog/sync-data-to-catalog/git/github/guides/connect-github-repository-with-sonarqube-project.md`)

### CODE REVIEW

**Comment by yairsimantov20** [comment]
i dont see it in the list of guides in here 
https://pr-1238.d2ngvl90zqbob8.amplifyapp.com/build-your-software-catalog/sync-data-to-catalog/code-quality-security/sonarqube/guides/connect-github-pr-with-sonar-analysis/

**Review by yairsimantov20 — CHANGES_REQUESTED** [review]
i dont see it in the list of guides in here
https://pr-1238.d2ngvl90zqbob8.amplifyapp.com/build-your-software-catalog/sync-data-to-catalog/code-quality-security/sonarqube/guides/connect-github-pr-with-sonar-analysis/

**Review by yairsimantov20 — APPROVED** [review]
🌊 Tech stuff looks good

File: docs/build-your-software-catalog/sync-data-to-catalog/git/github/guides/connect-github-repository-with-sonarqube-project.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import PortTooltip from "/src/components/tooltip/tooltip.jsx"
```
Comment (hadar-co) [inline docs/build-your-software-catalog/sync-data-to-catalog/git/github/guides/connect-github-repository-with-sonarqube-project.md:1]: @lordsarcastic The new "guides" folder has a lowercase "g" in the sidebar, it should be "Guides". To resolve this, please add a `_category_.json` file to the folder, see other folders for examples.

**Review by hadar-co — CHANGES_REQUESTED** [review]

**Review by hadar-co — APPROVED** [review]

**Pull request merged**